<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:rgb(192,192,192);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>542</width>
      <height>542</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>9</number>
    </property>
    <property name="columnCount">
     <number>9</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="6">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="7">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="8">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="5">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="6">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="7">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="8">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="5">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="6">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="7">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="8">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="5">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="6">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="7">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="8">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="5">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="6">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="7">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="8">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="5" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="5" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="5" column="5">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="5" column="6">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="5" column="7">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="5" column="8">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="6" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="6" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="6" column="5">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="6" column="6">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="6" column="7">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="6" column="8">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="7" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="7" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="7" column="5">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="7" column="6">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="7" column="7">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="7" column="8">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="8" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="8" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="8" column="5">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="8" column="6">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="8" column="7">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="8" column="8">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="random">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set map randomly</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manual">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set map manually</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="check">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>460</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Check answer</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>541</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>541</width>
      <height>10</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>541</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>10</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>10</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Black</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restart">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="solve">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>400</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Solve</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>    status</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="easy">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>170</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>easy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="medium">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>170</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>medium</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hard">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>200</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>hard</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
